app/lib/actions/katello/monitor_event_queue.rb

```diff
@@ -0,0 +1,128 @@
+module Actions
```

Comment:
For me, the triple, or quadrupel embedded classes makes it hard to parse what is going on and what belongs to what in this class (e.g. the `class << self` block).